crates/wash-lib/src/build/component.rs

```diff
@@ -201,18 +201,23 @@ async fn build_rust_component(
     // Change directory into the project directory
     std::env::set_current_dir(&common_config.path)?;
 
+    // Build for a specified target if provided, or the default rust target
+    let mut build_args = Vec::with_capacity(4);
```

Comment:
Nit: I think the `with_capacity` looks weird here because you dynamically push the release flag. I think `let mut build_args = vec!["build"];` would be simpler and better